### PR TITLE
PIM-5357: Add XLSX import/export asset, asset category, asset variation

### PR DESCRIPTION
**Pull request by SamirBoulil**
**Description**

Add XLSX job configuration for EE :
- import assets
- import asset categories
- export asset
- export asset categories
- export asset variation

Add PULL Request template with DOD

**Definition Of Done**

| Q | A |
| --- | --- |
| Added Specs | - |
| Generate fixtures and install | yes |
| Generate products and import | yes |
| Review and 2 GTM | yes |
|  Tech Doc | - |

### CODE REVIEW

**Comment by SamirBoulil** [comment]
@pierallard @aRn0D Shouldn't we add job accesses for EE jobs ? How does it work ?

**Comment by pierallard** [comment]
@SamirBoulil Nope, it's "automatic". It generates a file where everyone can do everything.

GTM when you removed "product" because you rollback it in master.

**Comment by nidup** [comment]
GTM

File: Resources/config/internal_jobs.yml [hunk 1]
```diff
@@ -144,76 +144,75 @@ jobs:
...
-        connector: Akeneo XLSX Connector
+        connector: Akeneo CSV Connector
         alias:     xlsx_association_type_import
         label:     Association type import
         type:      import
         configuration:
             uploadAllowed: true
     xlsx_association_type_export:
-        connector: Akeneo XLSX Connector
+        connector: Akeneo CSV Connector
```
Comment (momoss) [inline Resources/config/internal_jobs.yml]: It's XLSX...

**Pull request merged**